Misc/NEWS.d/next/Library/2022-01-25-15-31-04.bpo-46522.tYAlX4.rst

```diff
@@ -0,0 +1 @@
+fix concurrent.futures and io AttributeError messages
```

Comment:
Hmm, the NEWs is not clear.
Please describe better what is changed.

Reply:
```suggestion
make various module `__getattr__` AttributeErrors more closely match a typical AttributeError
```